backend/prisma/schema.prisma

```diff
@@ -102,6 +106,53 @@ enum InviteStatus {
   NOT_APPLICABLE
 }
 
+model Contact {
+  id                 String                    @id @default(uuid())
+  email              String?
+  firstName          String
+  lastName           String?
+  phoneNumber        String?
+  notes              String?
+  source             String?
+  ownerID            String
+  owner              User                      @relation(fields: [ownerID], references: [id])
+  importantDateEvent ImportantDateEvent[]
+  relationships      UserContactRelationship[]
+  Plan               Plan?                     @relation(fields: [planId], references: [id])
+  planId             String?
+}
+
+model ImportantDateEvent {
+  id        String  @id @default(uuid())
+  contact   Contact @relation(fields: [contactID], references: [id])
+  contactID String
+  date      String
+  eventType String
+}
+
+model UserContactRelationship {
+  id               String  @id @default(uuid())
+  contact          Contact @relation(fields: [contactID], references: [id])
+  contactID        String
+  relationshipType String
```

Comment:
should this be an enum?

Reply:
I'm not sure originally I had it as an enum but since I was getting confused on its implementation I just used String type field.

Reply:
We should use an enum instead of string here for better data integrity and to reduce errors

https://www.prisma.io/docs/concepts/components/prisma-schema/data-model#defining-enums

Reply:
okay I think I added the enum correctly now